docs/guide/exe_verilator.rst

```diff
@@ -1234,6 +1234,13 @@ Summary:
    :code:`/*verilator public_flat_rd*/`
    metacomments.
 
+.. option:: --public-ignore
+
+   Ignore all
+   :code:`/*verilator public* */`
+   metacomments. This is useful for optimized vpi builds, where a sim records reads/writes, then passes those with a .vlt file.
```

Comment:
vpi->VPI, sim->simulation.

Is below right?   We don't have an option to record read/writes - if that's a later pull update the docs then.

```suggestion
   metacomments. This is useful for speed-optimizing VPI builds where VPI is not being used.
```